batch/data-plane/src/test/java/com/microsoft/azure/batch/BatchIntegrationTestBase.java

```diff
@@ -1,52 +1,242 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
```

Comment:
License change is going in wrong direction?

Reply:
Updated the license header.